src/tests/profiler/rejit/rejit.cs

```diff
@@ -20,18 +29,41 @@ private static int MaxInlining()
 
             TriggerReJIT();
 
+            OutputBuilder.Clear();
+
             // TriggerInliningChain triggers a ReJIT, now this time we should call
             // in to the ReJITted code
             TriggerDirectInlining();
             CallMethodWithoutInlining();
             TriggerInliningChain();
 
+            string matchString = "Hello from profiler rejit method 'InlineeTarget'!";
+            int numRejittedTargets = OutputBuilder.ToString().Split(matchString).Length;
+            if (numRejittedTargets != 4)
+            {
+                Console.WriteLine("ReJIT did not update all instances of InlineeTarget!");
+                return 1234;
+            }
+
             TriggerRevert();
 
+            OutputBuilder.Clear();
+
             TriggerDirectInlining();
             CallMethodWithoutInlining();
             TriggerInliningChain();
 
+            if (OutputBuilder.ToString().Contains(matchString))
+            {
+                Console.WriteLine("ReJIT revert of InlineeTarget was not complete!");
+                return 1235;
+            }
+
+            // FIXME: Currently there will still be some 'Hello from profiler rejit method' messages left
```

Comment:
I doubt anyone will ever return to fix this so I'd remove the "FIXME" and just leave the rest of the comment describing current behavior. Alternately if you want to change it now I think all you'd need to do is add some filtering in GetReJITParameters() so that the test only modifies the IL when FunctionID == InlineeTarget.